R-packages/evalcast/.gitignore

```diff
@@ -1 +0,0 @@
-docs
```

Comment:
was this on purpose or did it get nuked in a rebuild?

Reply:
I'm not sure where that is. I think an early version had the docs folder buried in the package instead of alongside the covidcastR directory higher up.

Reply:
I think I mean that the former presence of `docs` in .gitignore implies that we didn't want these files in the repository. Has that changed? If you're not sure, then it would help to know the following:
* Are the files in the `docs` directory generated from source? 
* Are there any hand-edits we might want to maintain in the repository?
* Is there any other reason to retain the files in `docs` in the repository?

Reply:
Sorry, I should have been more clear. There is no `R-packages/evalcast/docs/` directory (nor should there be).